### ISSUE TITLE
Tearfulness

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#6783]
**Preferred term label:**
Tearfulness

**Synonyms**

**Definition (free text, please give PubMed ID)**

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
added
